### PR TITLE
i18n(es): Translated `statamic.mdx` 

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translated `statamic.mdx` to spanish 😄

### CODE REVIEW

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🚀, te dejo un par de sugerencias

**Comment by dreyfus92** [comment]
Gracias por el review @Waxer59 😁

File: src/content/docs/es/guides/cms/statamic.mdx [hunk 1]
```diff
@@ -0,0 +1,99 @@
+---
+title: Headless Statamic y Astro
+description: Agrega contenido a tu proyecto Astro usando Statamic como CMS
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/guides/cms/statamic.mdx:3]:
```suggestion
description: Agrega contenido a tu proyecto de Astro usando Statamic como CMS
```

File: src/content/docs/es/guides/cms/statamic.mdx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+Statamic viene con una [API REST](https://statamic.dev/rest-api) y una [API GraphQL](https://statamic.dev/graphql) integradas para conectar tus datos a Astro.
+
+### Prerrequisitos
+
+Para empezar, necesitarás lo siguiente:
+
+1. Solo están disponibles las API REST y GraphQL API en la versión pro de Statamic. Puedes probar la versión Pro gratis en tu [máquina local](https://statamic.dev/tips/how-to-enable-statamic-pro#trial-mode).
+2. **Un proyecto Astro** - Si todavía necesitas un proyecto Astro, nuestra [guía de instalación](/es/install/auto/) te pondrá en marcha rápidamente.
+3. **Un sitio Statamic** - Si necesitas un sitio Statamic, [esta guía](https://statamic.dev/quick-start-guide) te ayudará a empezar.
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/guides/cms/statamic.mdx:22]:
```suggestion
3. **Un sitio Statamic** - Si necesitas un sitio Statamic, [la guía de instalación de Statamic](https://statamic.dev/quick-start-guide) te ayudará a empezar.
```

File: src/content/docs/es/guides/cms/statamic.mdx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+Recuerda [habilitar la API REST](https://statamic.dev/rest-api#enable-the-api) o la [API GraphQL](https://statamic.dev/graphql#enable-graphql) añadiendo `STATAMIC_API_ENABLED=true` o `STATAMIC_GRAPHQL_ENABLED=true` en el archivo `.env` y habilitando los recursos necesarios en el archivo de configuración de la API.
+
+:::caution
+Todos los ejemplos asumen que tu sitio web tiene una colección llamada `posts`, que tiene un blueprint llamado `post`, y este blueprint tiene un campo de título (tipo de campo texto) y contenido (tipo de campo markdown).
+:::
+
+### Obteniendo Datos
+:::caution
+Si estás usando Statamic y Astro en tu máquina local, recuerda usar `127.0.0.1` en lugar de `localhost` al obtener datos de la API.
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/guides/cms/statamic.mdx:31]:
```suggestion
Si estás usando Statamic y Astro en tu máquina local, recuerda usar `127.0.0.1` en lugar de `localhost` al solicitar los datos a la API.
```

File: src/content/docs/es/guides/cms/statamic.mdx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+Cuando se solicita desde el servidor Astro, `localhost` no se resuelve correctamente como lo hace en el navegador y cualquier solicitud a cualquiera de las API fallará.
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/guides/cms/statamic.mdx:33]:
```suggestion
Cuando se solicita los datos desde el frontmatter de Astro, `localhost` no se resuelve correctamente como lo hace en el navegador y cualquier solicitud a cualquiera de las API fallará.
```

**Pull request merged**